charts/meilisearch/templates/tests/test-connection.yaml

```
@@ -14,5 +14,5 @@ spec:
     - name: wget
       image: busybox
       command: ['wget']
-      args:  ['{{ include "meilisearch.fullname" . }}:{{ .Values.service.port }}']
+      args:  ['--spider', '--timeout=5', '{{ include "meilisearch.fullname" . }}:{{ .Values.service.port }}']
```

YAMLlint (1.38.0)

[warning] too many spaces after colon

(colons)